book/src/pong-tutorial/pong-tutorial-05.md

```diff
@@ -844,8 +845,8 @@ handles to the `UiText` components that we want. Finally, we add the
 We're using `Write` here to pull in the `ScoreBoard` instead of with
 `WriteStorage` because we want mutable access to `ScoreBoard`, which is not a
 collection of components but rather a single resource item. This item is
-strictly required in all cases, but if it we didn't want it to always be
-required to exist, we could use `Option<Write<'s, ScoreBoard>>`.
+strictly required in all cases, but if we required it only for some cases
```

Comment:
"[..], but if we wanted it to be optional we could use ..."?

Reply:
I'm... not sure? But that would be more suitable.

Reply:
I'll edit with your suggestion for the moment :slightly_smiling_face: